cogboard-webapp/src/components/SnackbarWithVariant.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import ErrorIcon from '@material-ui/icons/Error';
+import InfoIcon from '@material-ui/icons/Info';
+import CloseIcon from '@material-ui/icons/Close';
+import {amber, blue, green} from '@material-ui/core/colors';
+import IconButton from '@material-ui/core/IconButton';
```

Comment:
import icons at once like you did with colors

Reply:
can't import different icons inline as they come from different libraries, e.g. **CloseIcon** is imported from _@material-ui/icons/**Close**_ but **InfoIcon ** is imported from _@material-ui/icons/**Info**_ etc.

Reply:
Yes you can. Check this out:
`import { SkipPrevious, PlayArrow, SkipNext } from '@material-ui/icons';`

Reply:
according to material-ui docs, the approach I used is safer, but ok, I will change that :D (https://material-ui.com/components/icons/)